pkg/cloud/gcp/actuators/machine/actuator.go

```diff
@@ -47,8 +47,16 @@ func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machi
 }
 
 func (a *Actuator) Exists(ctx context.Context, cluster *clusterv1.Cluster, machine *machinev1.Machine) (bool, error) {
-	// TODO(alberto): implement this
-	return false, nil
+	scope, err := newMachineScope(machineScopeParams{
+		machineClient: a.machineClient,
+		coreClient:    a.coreClient,
+		machine:       machine,
+	})
+	if err != nil {
+		return false, fmt.Errorf("failed to create scope for machine %q: %v", machine.Name, err)
+	}
+	defer scope.Close()
```

Comment:
How does `newReconciler` work with `scope` after it is closed on return from this function?

Reply:
Discussed this offline with Alberto. Fine for now.